src/constants/index.js

```diff
@@ -1,7 +1,7 @@
 // API
 export const API_URL = 'https://api.themoviedb.org/3'
 export const API_KEY = '3b00751f6f48427512666892cf2ba5fa'
-export const ALL_TRENDING_DAY_URL = '/trending/all/day'
+export const ALL_TRENDING_DAY_URL = '/trending/movie/day'
```

Comment:
TRENDING_MOVIE_DAY

Reply:
will be fixed in the next PR

Reply:
in real projects, no one will allow you. the merged code should have no problem